LDAR_Sim/src/virtual_world/nonfugitive_emissions.py

```diff
@@ -89,9 +94,9 @@ def estimate_start_date(self, cur_date: date, t_since_ldar: int) -> None:
             t_since_ldar (int): THe time in days since the site at which the emissions
             was discovered last received LDAR
         """
-        half_duration: int = ceil((t_since_ldar / 2))
-        self._estimated_days_active = half_duration
-        self._estimated_date_began = cur_date - timedelta(days=half_duration)
+        duration: int = t_since_ldar
```

Comment:
Kept because we plan to implement a user definable parameter to determine the assumption on duration - if we assume anything other full duration, we need to do some computation